.github/workflows/surge.yml

```diff
@@ -38,11 +38,11 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-unstable
 
-      - uses: cachix/cachix-action@v12
-        with:
-          name: pfhub-surge
-          extraPullNames: nix-community
-          skipPush: true
+      # - uses: cachix/cachix-action@v12
```

Comment:
Temporarily stopping cachix, or a permanent removal?

Reply:
Removed it now